subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -288,13 +288,65 @@ You can also assign collections or arrays of dependency notations to a configura
 
+Rendering the dependency tree is particularly useful if you'd like identify which dependencies have been resolved at runtime. It also provides you with information about any dependency conflict resolution that occurred in the process and clearly indicates the selected version. The dependency report always contains declared and transitive dependencies.
+
+Let's say you'd want to create tasks for your project that use the link:https://www.eclipse.org/jgit/[JGit library] to execute SCM operations e.g. to model a release process. You can declare dependencies for any external tooling with the help of a <<sub:scope_of_dependency_configurations,custom configuration>> so that it doesn't doesn't convolute other contexts like the compilation classpath for your production source code. To render the dependencies graph execute the `dependencies` task for the corresponding configuration.
+
+++++
+<sample id="dependencyReport" dir="userguide/dependencies/dependenciesReport" title="Rendering the dependency report for a custom configuration">
+    <sourcefile file="build.gradle" snippet="dependency-declaration" />
+    <output args="-q dependencies --configuration scm"/>
+</sample>
+++++
+
+[NOTE]
+====
+The dependencies report provides detailed information about the dependencies available in the graph. Any dependency that could not be resolved is marked with `FAILED` in red color. Dependencies with the same coordinates that can occur multiple times in the graph are omitted indicated by an asterisk (`*`) character. Dependencies that had to undergo conflict resolution render the requested and selected version separated by a right arrow character.
+====
+
+A link:https://scans.gradle.com/[build scan] can visualize dependencies as a navigable, searchable tree. Additional context information can be rendered by clicking on a specific dependency in the graph.
+
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>Dependency tree in a build scan</title>
+    <imageobject>
+        <imagedata fileref="img/dependency-management-dependencies-report-build-scan.png" width="180mm" />
+    </imageobject>
+</figure>
++++++
+
+==== Identifying the origin and selection reason for a dependency
+
+Large software projects inevitably deal with an increased number of dependencies either through direct or transitive dependencies. The <<sec:listing_dependencies,dependencies report>> provides you with the raw list of dependencies but does not explain _why_ they have been selected or _which_ dependency is responsible for pulling them into the graph.
+
+The _dependency insight report_ can answer both questions. Given a dependency in the dependency graph you can identify the selection reason and track down the origin of the dependency selection. You can think of the dependency insight report as the inverse representation of the dependency report for a given dependency.
```

Comment:
"Origin" in this context is a bit confusing to me, as I think of it as a sort of "network origin" which it's not. Perhaps a different word here and below.

Reply:
Maybe "root node"? Also confusing. Can't come up with something better right now.

Reply:
Does it mean "configuration"? Maybe you would explain it to me.

Reply:
It could be mean the declared dependency for a configuration or it could mean that it is coming from a transitive dependency which is pull in by a declared dependency through a configuration. Origin == whatever pulled it in.